packages/forklift-console-plugin/src/modules/Overview/utils/helpers/OverviewUserSettings.ts

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import React from 'react';
+
+import { loadFromLocalStorage, removeFromLocalStorage, saveToLocalStorage } from '@kubev2v/common';
+
+export interface OverviewUserSettings {
+  welcome?: WelcomeSettings;
+}
+
+interface WelcomeSettings {
+  hideWelcome: boolean;
+  save: (showWelcome: boolean) => void;
+  clear: () => void;
```

Comment:
save and clear are not `settings`, what am i missing ?

Reply:
Save to local storage and clear from local storage.

I used the same naming conversions as Radek used for the Pagination user settings, so that we will be able to unify it to one user settings component in the future:
https://github.com/kubev2v/forklift-console-plugin/blob/a8ced9a2617209f009f98a44915fbc7e2c28252d/packages/common/src/components/Page/types.ts#L1